tools/nl/svindex_differ/queryset_vars.csv

```diff
@@ -30,6 +30,83 @@ emit greenhouse gases
 cheapest houses
 show population
 
+#
+# From the list of queries on the frontend as suggestions.
+#
+Query

```

Comment:
drop "Query"?

Also, should we drop places?  (has some chance of polluting SV) rest of SVs don't have places

Reply:
Coming up in the follow up PR